specifications/api/swagger.yaml

```diff
@@ -461,7 +461,6 @@ definitions:
       - balance
       - delegation
       - name
-      - passphrase
```

Comment:
Just a question. Does this mean the swagger spec says that's it ok to create a wallet without a passphrase?

Reply:
It's even stronger than that. It means that ultimately, it's okay to create a wallet without a root private key. We'll come to that soonish.